core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -95,6 +96,9 @@ public int compare(final Transaction tx1, final Transaction tx2) {
     };
     private static final Logger log = LoggerFactory.getLogger(Transaction.class);
 
+    /** When this bit is set in protocolVersion, do not include witness */
```

Comment:
I'm trying to find this (and the described logic) in the specs, but did not find. Can you point me to it?

Reply:
This is an internal serialisation flag that bitcoin core uses, I don't think it is in the specs. I just used the same for consistency.

https://github.com/bitcoin/bitcoin/blob/0.19/src/primitives/transaction.h#L15

Reply:
Ok, then I think it would be a good idea to explain in the comment that this quirk was directly ported from Bitcoin Core.

Reply:
Done